amqp_influxdb/__init__.py

```diff
@@ -44,6 +45,24 @@ def __init__(self,
 
         connection_parameters = connection_parameters or {}
 
+        if isinstance(connection_parameters.get('credentials'), dict):
+            credentials = connection_parameters['credentials']
+            credentials_object = pika.credentials.PlainCredentials(
+                username=credentials['username'],
+                password=credentials['password'],
+            )
+            connection_parameters['credentials'] = credentials_object
+
+        if connection_parameters['ca_path'] != '':
```

Comment:
I prefer to be explicit rather than rely on False-yness of different objects in python. Can be changed if strongly preferred though.